python/cudf/dataframe/series.py

```diff
@@ -1113,6 +1113,40 @@ def product(self, axis=None, skipna=True, dtype=None):
         assert axis in (None, 0) and skipna is True
         return self._column.product(dtype=dtype)
 
+    def cummin(self, axis=0, skipna=True):
+        """Compute the cumulative minimum of the series"""
+        assert axis in (None, 0) and skipna is True
+        return Series(self._column._apply_scan_op('min'), name=self.name)
```

Comment:
You're dropping the index in this operation, unsure if that's intended.

Reply:
Thanks. Oversight from late at night.